backend/config/admin.js

```diff
@@ -0,0 +1,5 @@
+module.exports = ({ env }) => ({
+  auth: {
+    secret: env('ADMIN_JWT_SECRET', 'd8ea0029965dfd06dfd92608b1ee0eff'),
```

Comment:
This is the default right? I know this question has come up a few times before.

Reply:
Yes that's autogenerated. I'll create a task to remove the default value so we are forced to set the env variable 👍